director/users/admin.py

```diff
@@ -0,0 +1,62 @@
+from typing import Set
+
+from django.contrib import admin
+from django.contrib.auth.models import User
+from django.contrib.auth.admin import UserAdmin
+
+# Unregister the default user model admin
+admin.site.unregister(User)
+
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    """A custom admin interface for User instances."""
+
+    readonly_fields = [
+        'last_login',
+        'date_joined',
+    ]
+
+    def get_form(self, request, obj=None, **kwargs):
+        """
+        Restrict the fields that staff can change.
+
+        For more on the why and how of this see https://realpython.com/manage-users-in-django-admin/
+        """
+        form = super().get_form(request, obj, **kwargs)
+        is_superuser = request.user.is_superuser
+        disabled_fields = set()  # type: Set[str]
```

Comment:
Since we use real type annotations we don't have to have this fake "in comment ones", so this can be:
`disabled_fields: Set[str] = set()`

Reply:
Thanks for the clarification. Did a copy pasta on that one.